From aa511f8856b4a437de005f23c6932948fffd8768 Mon Sep 17 00:00:00 2001
From: Alberto Leiva Popper <ydahhrk@gmail.com>
Date: Sat, 1 Jul 2023 09:52:09 -0600
Subject: [PATCH] Include csum_tcpudp_magic's header

Fixes build in some kernels.
In fact, I'm not sure why I'm not getting the warning. It looks as
though I should.

Thanks to Charles Hardin for this patch.

Fixes #407.
---
 src/mod/common/rfc7915/6to4.c | 1 +
 1 file changed, 1 insertion(+)

--- a/src/mod/common/rfc7915/6to4.c
+++ b/src/mod/common/rfc7915/6to4.c
@@ -1,6 +1,7 @@
 #include "mod/common/rfc7915/6to4.h"
 
 #include <linux/inetdevice.h>
+#include <net/ip6_checksum.h>
 #include <net/udp.h>
 #include <net/tcp.h>
 
